utils/find-overlay-dependencies.sh

```diff
@@ -68,16 +68,22 @@ for sdk in ${(k)SDKS}; do
   DEPENDS_ON=()
   printf "%s:\n\t" "$sdk"
   deps=$(echo "@import $1;" | xcrun -sdk $sdk clang -arch $arch -x objective-c - -M -fmodules 2>/dev/null)
+  if [[ $? != 0 ]]; then
+    # Don't `continue` here because we want to clear out the SDK line if it exists
```

Comment:
I'm not sure this is correct. We'd probably want to *warn* on such a line rather than clear it out. There aren't actually any modules with no dependencies except Darwin.